include/knowhere/prometheus_client.h

```diff
@@ -55,22 +55,35 @@ class PrometheusClient {
 extern const prometheus::Histogram::BucketBoundaries defaultBuckets;
 extern const std::unique_ptr<PrometheusClient> prometheusClient;
 
-#define DEFINE_PROMETHEUS_GAUGE(name, desc)                                                                  \
-    prometheus::Family<prometheus::Gauge>& name##_family =                                                   \
-        prometheus::BuildGauge().Name(#name).Help(desc).Register(knowhere::prometheusClient->GetRegistry()); \
+#define KNOWHERE_LABEL "knowhere"
+
+#define DEFINE_PROMETHEUS_GAUGE(name, desc, label)                                                                   \
+    prometheus::Family<prometheus::Gauge>& name##_family = prometheus::BuildGauge()                                  \
```

Comment:
it seems to me a bad idea to create a new family for every new metric? for example can we have a latency family and create multiple gauges in this family for search/build/load latency respectively.